Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -178,9 +178,32 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
     // MARK: - Card Delegate
 
     func card(_ cardClient: CardClient, didFinishWithResult result: CardResult) {
-        updateTitle("Order Id:\(result.orderID) status: \(result.status)\n \(String(describing: result.paymentSource))")
+        let intent = DemoSettings.intent.rawValue.uppercased()
+        if intent == "CAPTURE" {
+            updateTitle("Capturing Order with Id:\(result.orderID)...")
```

Comment:
nit as iOS prefers `ID` over `Id`: 
```suggestion
            updateTitle("Capturing Order with ID:\(result.orderID)...")
```